src/components/AdminChart/AdminChart.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react'
+import Chart from 'react-apexcharts'
+import { useDispatch } from 'react-redux'
+import { getIncomeByMonth } from '../../page-components/Admin/admin.slice'
+import { unwrapResult } from '@reduxjs/toolkit'
+
+export default function AdminChart() {
+  const dispatch = useDispatch()
+  let dataChart = []
+  const [incomeByMonth, setIncomeByMonth] = useState([])
+
+  useEffect(() => {
+    const _getIncomeByMonth = async () => {
+      const res = await dispatch(getIncomeByMonth())
+      const data = unwrapResult(res)
+      setIncomeByMonth(data.data)
+    }
+    _getIncomeByMonth()
+  }, [dispatch])
+
+  for (let i = 0; i < 12; i++) dataChart[i] = 0
+  incomeByMonth.length &&
+    incomeByMonth.forEach(value => {
+      if (value.year === 2022) {
+        dataChart[value.month - 1] = value.totalSaleAmount
+      }
+    })
+
+  const chartOptions = {
+    options: {
```

Comment:
```suggestion
  const chartOptions = {
```
đâu cần thằng này làm gì đâu nhỉ

Reply:
em đã fix